pkgs/by-name/ne/netbeans/package.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchurl, makeWrapper, makeDesktopItem, which, unzip, libicns, imagemagick
-, jdk, perl
+, jdk17, perl
```

Comment:
We might be able to bump this to jdk21. [Per the download page](https://netbeans.apache.org/front/main/download/nb21/index.html):

> The Apache NetBeans 21 binary releases require JDK 11+, and officially support running on JDK 11, 17 and 21.